runtime/syntax/cpp.yaml

```diff
@@ -47,9 +47,9 @@ rules:
     - constant.string:
         start: "'"
         end: "'"
-        skip: "\\\\."
+        skip: "(\\\\.)|(\\b[1-9][0-9]*'[0-9']*[0-9]|0[0-7]+'[0-7']*[0-7]|0[Xx][0-9A-Fa-f]+'[0-9A-Fa-f']*[0-9A-Fa-f]|0[Bb][01]+'[01']*[01]\\b)"
         rules:
-            - error: "..+"
+            - error: "[[:graph:]]{2,}'"
```

Comment:
@dmaluka 
Could you confirm this again? `- error: "[[:graph:]]{2,}'"` is highlighting `'ab'` for me.

The whole point of this change is because `"..+"`  is not working properly with
```cpp
const std::unordered_set<char> CharTokens = {'(', ')', ';', '{', '}', '/', '#', ',', '=', '<', '>'};
```

I have not tried the change in #3127 but if that indeed fixes, I don't mind going back to `"..+"`
But that means this PR would need to be merged after it though otherwise the line I mentioned above will be highlighted as error

Reply:
Ok, I've double-checked: with your PR as is, `'ab'` is correctly highlighted as error. But with your PR applied on top of #3127 it is not. And then, after I replace your `error: "[[:graph:]]{2,}'"` with the original `error: "..+"`, it is correctly highlighted as error again.

I think this `"[[:graph:]]{2,}'"` is obviously incorrect (the error regex should not include the single quote, since the single quote is not a part of the string that is highlighted as an error), and happens to somehow work correctly just because, as I said, there is a bug in micro.